app/helpers/application_helper.rb

```diff
@@ -2,6 +2,6 @@
 
 module ApplicationHelper
   def active_class(link_path)
-    current_page?(link_path) ? 'active' : ''
+    return 'active' unless current_page?(link_path)
```

Comment:
1. I think you got this backwards. They should have the 'active' class _if_ this is the current page, not _unless_ it is.
2. Very minor nit-pick (perhaps appropriate here, ahem): You can forgo the explicit return.

So, this should be `'active' if current_page?(link_path)`

Reply:
Thanks!